src/components/activities/ActivityCard/QuestionCard.tsx

```diff
@@ -31,7 +31,7 @@ export const QuestionCard = ({ activity, noButtons, showEditButtons, onDelete }:
     } else {
       askSame.push(user.id);
     }
-    await updatedActivityData(activity, {
+    await askSameQuestion(activity, {
```

Comment:
There's no need anymore to update askSame here, the backend handle it now.
So you can do:

```tsx
const onAskSame = async () => {
  if (!user || isSelf) {
    return;
  }
  await askSameQuestion(activity.id);
}
```